microsoft-edge/dualengine/reference/tagsentinelentryinfo.md

```diff
@@ -0,0 +1,48 @@
+---
+description: Represents the position of the current region in the travel log.
+title: DualEngine Win32 C++ tagSentinelEntryInfo

```

Comment:
seems would be a verbose title - check rendered: 
https://review.learn.microsoft.com/en-us/microsoft-edge/dualengine/reference/tagsentinelentryinfo?branch=pr-en-us-2859
oh, this title is for the tab.  ok?

Reply:
```
title: DualEngine Win32 C++ tagSentinelEntryInfo
...
# struct tagSentinelEntryInfo
```
not sure if this disparity between tab title vs. top-of-article title is ok in Ref topics.  Might be ok in this context. These strings are identical in conceptual .md files.  @AndrewLakeMSFT

Reply:
these titles match the same format as the webview2 reference pages